packages/client/tests/functional/_utils/setupTestSuiteMatrix.ts

```diff
@@ -144,6 +144,11 @@ function setupTestSuiteMatrix(
             // sometimes we test connection errors. In that case,
             // disconnect might also fail, so ignoring the error here
           })
+
+          if (client._adapter) {
```

Comment:
Can we reuse the link we can from `newDriverAdapter` here, rather than add a property to a client that would be used only in test?

Reply:
Actually, I will try removing the `client._adapter.close()` altogether, in the spirit of [this comment of yours](https://prisma-company.slack.com/archives/C04TW4A2H8C/p1699354430085669?thread_ts=1699349717.575299&cid=C04TW4A2H8C), and see whether all latent connections are actually released implicitly.